src/components/messageCanvas/messageCanvas.tsx

```diff
@@ -49,7 +49,7 @@ function MessageCanvasElement(
       <Stack direction="row" alignItems="center" spacing={1}>
         {props.getProfileComponent && props.getProfileComponent(props.message)}
         <Typography variant="body1" color="text.secondary" data-cy="sender">
-          {props.message.sender}
+          {props.message.sender.name}
```

Comment:
If name is not available, do we want to show sender id instead?

Reply:
`id` might not be useful for ppl. How about we leave it on the profile component to decide what to display ?